File: src/main/java/com/ssafy/ChallenMungs/challenge/panel/repository/PanelRepository.java
package com.ssafy.ChallenMungs.challenge.panel.repository;

import com.ssafy.ChallenMungs.challenge.general.entity.General;
import com.ssafy.ChallenMungs.challenge.panel.entity.Panel;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.data.querydsl.QuerydslPredicateExecutor;

public interface PanelRepository extends JpaRepository<Panel, Long>, QuerydslPredicateExecutor<Panel> {
}

File: ChallenMungs/app/src/main/java/com/ssafy/challenmungs/domain/repository/PanelRepository.kt
package com.ssafy.challenmungs.domain.repository

import com.ssafy.challenmungs.data.remote.Resource
import com.ssafy.challenmungs.domain.entity.challenge.ChallengeInfo

interface PanelRepository {

    suspend fun requestChallengeInfo(challengeId:Long):Resource<ChallengeInfo>
}